package dispatch;

message Request {
	optional string ctx = 1;
	optional string account = 2;
	optional string client_ip = 3;
	optional int32 client_port = 4;
}

message Reply {
	optional int32	result = 1;
	optional string server_ip = 2;
	optional int32	server_port = 3;
	optional string client_ip = 4;
	optional int32 client_port = 5;
}

